src/incentives/package.json

```diff
@@ -13,7 +13,7 @@
         "prod-pc": "SET NODE_ENV=production&&nodemon ./bin",
         "stage-pc": "SET NODE_ENV=staging&&nodemon ./bin",
         "prettier": "prettier --single-quote --print-width 80 --trailing-comma all --write 'src/**/*.js'",
-        "test": "mocha ./**/test/ut_*.js --exit"
+        "test": "nyc --reporter=lcov mocha ./**/test/ut_*.js --exit"
```

Comment:
@Psalmz777 , I am still noticing different styles here. Care to shed more light here? Noticed another configuration where you introduced a new command of test-cov or something? That PR was for the data-mgt microservice and it has just been merged.

Reply:
@Martin it is essentially the same thing ,I worked on data-mgt first so I introduced a new script to cover for the test-coverage , while I was working on incentives though I noticed that I could append the test coverage in the same script as the test this the difference,as for the test cov on data-mgt , I hadn't worked on mocha tests before so it was one of the first methods I landed on but I noticed I could do it better that's why the test-cov isn't there in the final draft